### PR TITLE
Add FAQ entry to explain module adoptions #2431

### PR DESCRIPTION
**Pull request by szabgab**

### CODE REVIEW

File: root/about/faq.html [hunk 1]
```diff
@@ -151,6 +151,42 @@
...
+
+### How to become the maintainer or co-maintainer of a CPAN distribution?
+
+* TBD.
+
+### How to mark a distribution that is up for adoption?
+
+If you are the maintainer of the distribution you can follow these steps:
+
+* Visit <a href="https://pause.perl.org/">PAUSE</a>.
```
Comment (haarg) [inline root/about/faq.html:171]: Probably better to use markdown linking here.
```
* Visit [PAUSE](https://pause.perl.org/).
```

File: root/about/faq.html [hunk 2]
```diff
@@ -151,6 +151,42 @@
...
+* Check the box next to the name of the distribution.
+* Type in HANDOFF as userid.
+* Click on "Make Co-Maintainer"
+* TBD: How long till the change shows up on MetaCPAN?
+
+### How to remove the "adopt me" banner from the module?
+
+Remove the author ADOPTME, HANDOFF, or NEEDHELP from the project.
+
+* Visit <a href="https://pause.perl.org/">PAUSE</a>.
```
Comment (haarg) [inline root/about/faq.html:183]: Same as above.

File: root/about/faq.html [hunk 3]
```diff
@@ -151,6 +151,42 @@
...
+### What does the "Take me over?", "Why not adopt me?", "Looking for help!" mean?
+
+CPAN Distributions can be marked by their authors, or by the CPAN admins, in case the author is not
+available to be "up for adoption". This can be done by adding either one of the following PAUSE accounts
+as co-maintainers: ADOPTME, HANDOFF, or NEEDHELP.
+
+
+### How to become the maintainer or co-maintainer of a CPAN distribution?
+
+* TBD.
```
Comment (oalders) [inline root/about/faq.html:165]: This section could include a link to https://neilb.org/adoption/

**Comment by oalders** [comment]
@neilb do you have any blog posts that would be relevant to link to here?

File: root/about/faq.html [hunk 4]
```diff
@@ -151,6 +151,42 @@
...
+
+If you are the maintainer of the distribution you can follow these steps:
+
+* Visit <a href="https://pause.perl.org/">PAUSE</a>.
+* Click on "Add Comaintainers"
+* Filter for the name of the distribution.
+* Check the box next to the name of the distribution.
+* Type in HANDOFF as userid.
+* Click on "Make Co-Maintainer"
+* TBD: How long till the change shows up on MetaCPAN?
```
Comment (oalders) [inline root/about/faq.html:179]: This runs on an hourly cron.

File: root/about/faq.html [hunk 5]
```diff
@@ -151,6 +151,42 @@
...
+### How to remove the "adopt me" banner from the module?
+
+Remove the author ADOPTME, HANDOFF, or NEEDHELP from the project.
+
+* Visit <a href="https://pause.perl.org/">PAUSE</a>.
+* Click on "Remove Comaintainers".
+* Filter for the name of the distribution.
+* Check which one of the above 3 are listed.
+* Select them and click on "Remove".
+* TBD: How long till the change shows up on MetaCPAN?
```
Comment (oalders) [inline root/about/faq.html:190]: This runs on an hourly cron.

File: root/about/faq.html (hunk 3 above)
Comment (oalders) [reply root/about/faq.html:166]: I think the "TBD" is the only thing left to clarify here.

Comment (szabgab) [reply root/about/faq.html:166]: Is there a blog post explaining I could link to?

Comment (oalders) [reply root/about/faq.html:166]: How about http://neilb.org/2013/07/24/adopt-a-module.html ?

**Comment by neilb** [comment]
Could also link to http://neilb.org/2014/01/31/adoption-request.html which is a template mail message if you want to email someone asking to adopt their module, but the wording of the template suggests a bunch of things you should have done before you're emailing someone asking to take over their dist.

**Review by oalders — APPROVED** [review]
Thanks for this @szabgab! Let's go live with it and see what feedback we get.

**Comment by szabgab** [comment]
Thank you. When is this expected to be deployed?

**Comment by oalders** [comment]
This will have already been deployed. Deployments happen every 30 or 60 minutes. (Can't remember which).

You can purge the cache for any individual page via 

```
curl -XPURGE https://metacpan.org/path/to/page
```

If you can't see your changes immediately after a purge, please let me know.

**Comment by szabgab** [comment]
Thanks. That worked and I can see the new content, but I don't understand why do we need to do this manually?

**Comment by oalders** [comment]
The caching is done via Fastly and our deployments are not sophisticated enough to pick up all the correct URLs to purge on arbitrary code merges. We do catch some cases, but not all of them. They'll eventually expire, so it's generally not a huge issue, but we occasionally purge some URLs manually or just purge everything.

**Pull request merged**